setup.cfg

```diff
@@ -23,6 +23,7 @@ packages = find:
 install_requires =
     auditree-framework>=1.2.3
     auditree-harvest>=1.0.0
+    PyYAML
```

Comment:
This doesn't appear to be used in this PR?

Reply:
Thank you, it was used to read kubeconfig files in the older version of cluster resource fetcher, but it is no longer used.  I removed this line.